sqlglot/dialects/snowflake.py

```diff
@@ -327,3 +327,10 @@ def generatedasidentitycolumnconstraint_sql(
             increment = expression.args.get("increment")
             increment = f" INCREMENT {increment}" if increment else ""
             return f"AUTOINCREMENT{start}{increment}"
+
+        def create_sql(self, expression: exp.Create) -> str:
```

Comment:
delete this and make a new propety location unsupported, then snowflake just needs to override the properties_location